Automattic/wp-calypso
# Signup: Add squares to background

#### Changes proposed in this Pull Request

* Alternate title: Just the squares, m'am
* Trying another approach (similar to #27938) in just adding the background decoration to signup for a bit of visual interest.

Before:

<img width="1440" alt="screen shot 2018-10-24 at 12 21 06 pm" src="https://user-images.githubusercontent.com/2124984/47445601-51ef6380-d787-11e8-8188-3a2e61793f29.png">


After: 

<img width="1440" alt="screen shot 2018-10-24 at 12 19 44 pm" src="https://user-images.githubusercontent.com/2124984/47445584-4603a180-d787-11e8-939c-7130c5b1ac28.png">

#### Testing instructions

* Switch to this PR, navigate to `/start`
* Check out the fancy squares!
